entrypoint/fb.js

```diff
@@ -178,3 +179,21 @@ module.exports.push = (event, context, callback = console.log) => {
     })
   })
 };
+
+module.exports.attachment = (event, context, callback) => {
+  const payload = JSON.parse(event.body);
+  console.log(payload);
```

Comment:
Is this logging still necessary or was it just for testing?

Reply:
I'd prefer to keep it